contracts/hardhat.config.ts

```diff
@@ -47,8 +47,16 @@ const config: HardhatUserConfig = {
           evmVersion: "cancun",
         },
       },
+      /**
+       * Maintain for Verifier contracts
+       * src/_testing/unit/verifiers/TestPlonkVerifierForDataAggregation.sol
+       * src/verifiers/PlonkVerifierDev.sol
+       * src/verifiers/PlonkVerifierForDataAggregation.sol
+       * src/verifiers/PlonkVerifierForMultiTypeDataAggregation.sol
+       * src/verifiers/PlonkVerifierSepoliaFull.sol
+       */
       {
-        version: "0.8.25",
+        version: "0.8.26",
```

Comment:
we can bump these, we just need to update the autoupdate.sh script to `sed` the correct version

Reply:
fixed